packages/contracts/test/diamond/facets/CurveDollarIncentiveFacet.t.sol

```diff
@@ -108,58 +102,51 @@ contract CurveDollarIncentiveTest is DiamondSetup {
             mockReceiver,
             false
         );
-        ICurveDollarIncentiveFacet.switchBuyIncentive();
         vm.stopPrank();
 
         vm.prank(mockDollarManager);
         ICurveDollarIncentiveFacet.incentivize(
             stableSwapPoolAddress,
             mockReceiver,
-            amountIn
+            100e18
         );
 
         last_balance = governanceToken.balanceOf(mockReceiver);
         assertEq(last_balance, init_balance);
 
-        // // 3. do nothing if no incentive
-        mockInternalFuncs(1e18);
-        init_balance = governanceToken.balanceOf(mockReceiver);
-        vm.startPrank(admin);
-        ICurveDollarIncentiveFacet.setExemptAddress(
-            mockReceiver,
-            false
-        );
-        ICurveDollarIncentiveFacet.switchBuyIncentive();
-        vm.stopPrank();
-
-        vm.prank(mockDollarManager);
-        // ICurveDollarIncentiveFacet.incentivize(
-        //     stableSwapPoolAddress,
+        // 3. do nothing if no incentive
```

Comment:
you can do better ;) check how I mocked the twap inside twapOracleFacet

Reply:
the goal is to add a mock for twap oracle like I did in the twapOracleFacet tests so that the test stays the same as before